trustregistry/registry/registry_actors.py

```diff
@@ -57,11 +59,11 @@ async def update_actor(actor_id: str, actor: Actor, db: Session = Depends(get_db
 
 
 @router.get("/did/{actor_did}")
-async def get_actor_by_did(actor_did: str, db: Session = Depends(get_db)):
+async def get_actor_by_did(actor_did: str, db_session: Session = Depends(get_db)):
     bound_logger = logger.bind(body={"actor_did": actor_did})
     bound_logger.info("GET request received: Get actor by DID")
     try:
-        actor = crud.get_actor_by_did(db, actor_did=actor_did)
+        actor = crud.get_actor_by_did(db_session, actor_did=actor_did)
     except crud.ActorDoesNotExistException:
         bound_logger.info("Bad request: Actor not found.")
         raise HTTPException(status_code=404, detail="Actor not found.")
```

Comment:
I think all of these "Actor not found" messages should be more informative, because some we're requesting by id, some by did. So it helps to tell the user "Actor with did {} not found"